content/en/docs/setup/additional-setup/external-istiod/index.md

```diff
@@ -0,0 +1,297 @@
+spec:
+ meshConfig:
+   rootNamespace: external-istiod
+   defaultConfig:
+     discoveryAddress: $REMOTE_ISTIOD_ADDR:15012
+     proxyMetadata:
+       XDS_ROOT_CA: /etc/ssl/certs/ca-certificates.crt
+       CA_ROOT_CA: /etc/ssl/certs/ca-certificates.crt
+ components:
+   pilot:
+     enabled: false
+   istiodRemote:
+     enabled: true
+
+ values:
+   global:
+     caAddress: $REMOTE_ISTIOD_ADDR:15012
+     istioNamespace: external-istiod
+   istiodRemote:
+     injectionURL: https://$REMOTE_ISTIOD_ADDR:15017/inject
+   base:
+     validationURL: https://REMOTE_ISTIOD_ADDR:15017/validate
+EOF
+{{< /text >}}
+
+Install the configuration in `remote_cluster`:
+
+{{< text bash >}}
+$ istioctl apply -f remote-config-cluster.yaml --context="${CTX_REMOTE_CLUSTER}"
+{{< /text >}}
+
+You may notice the ingress gateway in `remote_cluster` is not running yet. This is expected until the external istiod is running, which you will install next.
+
+### Setup external istiod in the control plane cluster
+
+Create remote secret to allow external istiod in `external_cluster` to access the `remote_cluster`:
+
+{{< text bash >}}
+$ kubectl create sa istiod-service-account -n external-istiod --context="${CTX_EXTERNAL_CLUSTER}"
+$ istioctl x create-remote-secret \
+  --context="${CTX_REMOTE_CLUSTER}" \
+  --type=config \
+  --namespace=external-istiod | \
+  kubectl apply -f - --context="${CTX_EXTERNAL_CLUSTER}"
+{{< /text >}}
+
+Generate the Istio configuration for the `external-istiod` namespace in `external_cluster`:
+
+{{< text bash >}}
+$ cat <<EOF > external-istiod.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+ namespace: external-istiod
+spec:
+ meshConfig:
+   defaultConfig:
+     discoveryAddress: $REMOTE_ISTIOD_ADDR:15012
+     rootNamespace: external-istiod
+     proxyMetadata:
+       XDS_ROOT_CA: /etc/ssl/certs/ca-certificates.crt
+       CA_ROOT_CA: /etc/ssl/certs/ca-certificates.crt
+ components:
+   base:
+     enabled: false
+   ingressGateways:
+   - name: istio-ingressgateway
+     enabled: false
+ values:
+   global:
+     caAddress: $REMOTE_ISTIOD_ADDR:15012
+     istioNamespace: external-istiod
+     operatorManageWebhooks: true
+   pilot:
+     env:
+       INJECTION_WEBHOOK_CONFIG_NAME: ""
+       VALIDATION_WEBHOOK_CONFIG_NAME: ""
+EOF
+{{< /text >}}
+
+Apply the Istio configuration in `external_cluster`:
+
+{{< text bash >}}
+$ istioctl apply -f external-istiod.yaml --context="${CTX_EXTERNAL_CLUSTER}"
+{{< /text >}}
+
+### Validate the installation
+
+Confirm the Istio ingress gateway in `remote_cluster` is running.
+
+{{< text bash >}}
+$ kubectl get pod -l app=istio-ingressgateway -n external-istiod --context="${CTX_REMOTE_CLUSTER}"
+{{< /text >}}
+
+Deploy the sleep sample in `remote_cluster` with a namespace has [automatic sidecar injection](/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection) enabled.  The sleep pod should reach running in a few seconds.
```

Comment:
This isn't really much of a validation. Can't you at least run helloworld, and test that you can access it through the ingress gateway?

Reply:
This is the most critical testing we need :) because getting the sleep pod up running means we have sidecar injected from external istiod and able to establish the connection with external istiod. :)   

But I'd be happy to add a link to point to helloworld or something to test the gateway as well.

Reply:
addressed, I think!